rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
I will confirm with @masinoa regarding that, but those should be the *only* set of files that are required to be added to the container. The rest of the variables should be environment variables.